common/pii/Pii.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional, Generic, TypeVar, Sequence, Tuple
+
+from absl import flags
+
+FLAGS = flags.FLAGS
+flags.DEFINE_bool("collect_pii", False, "Whether to store PII.")
```

Comment:
I assume this won't override the user's own `collect_pii` setting set in arguments as `common.pii.Pii` is imported before args are parsed. 

Is this is just to ensure the default setting is False?

Reply:
Yeah, `False` is just the default.